pkg/metrics/metrics.go

```diff
@@ -25,4 +30,99 @@ var (
 			Help: "RHMI status available",
 		},
 	)
+
+	RHMIInfo = prometheus.NewGaugeVec(
+		prometheus.GaugeOpts{
+			Name: "rhmi_info",
+			Help: "RHMI info variables",
+		},
+		[]string{
+			"use_cluster_storage",
+			"master_url",
+			"installation_type",
+			"operator_name",
+			"namespace",
+		},
+	)
+
+	RHMIStatus *prometheus.GaugeVec = nil
+
+	rhmiStatusLabels = []string{
+		"operator_name",
+		"namespace",
+		"last_error",
+		"preflight_message",
+		"preflight_status",
+		"stage",
+	}
 )
+
+// SetRHMIInfo exposes rhmi info metrics with labels from the installation CR
+func SetRHMIInfo(installation *integreatlyv1alpha1.RHMI) {
+	RHMIInfo.WithLabelValues(installation.Spec.UseClusterStorage,
```

Comment:
There are additional fields included in the CR that might be useful here as well (namespacePrefix, operatorsInProductNamespace, routingSubdomain, selfSignedCerts). Unless the decision was made to only include these 5.

Reply:
David Martin suggested these fields but I don't see any problem in adding more, what do you think @david-martin ?

Reply:
The storage overhead tends to be with the values for each scrape time (lots of scrapes) rather than the labels, 
unless the labels change very frequently.
It should be safe, from a storage point of view, to include more labels as long as they don't change very often.

Reply:
Perfect, thanks @david-martin

Reply:
Cool, I don't think these values should ever change once the installation is completed, I'll add them to the metric.